app/models/search_builder.rb

```diff
@@ -12,5 +12,7 @@ class SearchBuilder < Blacklight::SearchBuilder
 
```

Comment:
Since I'm asking questions, what are the non-catalog search builders? Are they used in 'My works' and similar areas (my only guess, really)? Or how can I find out what they are?